### PR TITLE
Update technical-preview-free-tier.md

### PR DESCRIPTION
**Pull request by danieltprice**
Comment out the query that does not work:
https://websitemain62807-dpricedatasizecontentupdates.gtsb.io/docs/reference/technical-preview-free-tier/

### CODE REVIEW

**Review by knizhnik — COMMENTED** [review]
Sorry, I still do not understand why do we need to comment  usage of `pg_database_size()`.
If we limit logical size of the database, then definitely users should have some way to check the current size, otherwise limit exhaustion error will be  unpleasant surprise.

Concerning `pg_cluster_size` and `neon` extension - them are needed mostly for us. Their descriptoin can really be removed from documentation.

**Comment by danieltprice** [comment]
Thank you, @knizhnik. I misunderstood. I thought the first query was was also not ideal. I have added it back.

**Comment by stepashka** [comment]
let's move out the details about how limita are calculated out of the free tier article
the 'Reference' section may be a good place for that

**Review by stepashka — COMMENTED** [review]
let's move out the details about how limita are calculated out of the free tier article
the 'Reference' section may be a good place for that

**Comment by danieltprice** [comment]
> let's move out the details about how limita are calculated out of the free tier article the 'Reference' section may be a good place for that

File: content/docs/reference/technical-preview-free-tier.md [hunk 1]
```diff
@@ -18,18 +18,20 @@ Technical Preview Free Tier users can create up to three projects in Neon. Proje
 Neon separates storage and compute and stores data in its own internal format.
 The 10GB data size limit applies to the logical size of a Neon project. The logical size is the sum of all relation sizes in the project.
 
-For users familiar with PostgreSQL, the logical size is roughly equal to:
+To check the logical size of the relations in your Neon project, run the following query:
```
Comment (stepashka) [inline content/docs/reference/technical-preview-free-tier.md:21]: not sure every one of our users would be familiar with the word 'relations'

**Review by stepashka — APPROVED** [review]
not sure every one or even most of our users would be familiar with the word 'relations'

let's fix it soon, but not insisting on delaying this merge for that.

File: content/docs/reference/technical-preview-free-tier.md (hunk 1 above)
Comment (danieltprice) [reply content/docs/reference/technical-preview-free-tier.md:21]: Let's change that to databases!

**Review by stepashka — APPROVED** [review]

**Pull request merged**